crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -240,6 +240,44 @@ impl ColumnType {
         )
     }
 
+    /// Returns the number of bits in the integer type if it is an integer type. Otherwise, return None.
+    pub fn to_integer_bits(&self) -> Option<usize> {
```

Comment:
Not a big fan of this function. At the very least it should be private.

My preference would be for the `max_integer_type` function to just have a (admittedly tedious) match.
```suggestion
    fn to_integer_bits(&self) -> Option<usize> {
```

Reply:
I just made it private. I do think the match is not going to be great since we plan to add even more integer types.